scripts/tools/zap/README.md

```diff
@@ -0,0 +1,190 @@
+---
+orphan: true
+---
+
+# ZAP
+
+This directory contains various tools related to ZAP.
+
+# ZAP Cluster Revision Update Tool (update_cluster_revisions.py)
+
+This tool parses all ZAP files in the codebase and updates cluster revisions.
+
+**WARNING**: This tool only updates the revision number. Please ensure any new
+attributes, events or commands are implemented accordingly.
+
+## Prerequisites
+
+This tool uses the python environment, which can be built and activated using:
```

Comment:
Hmm.  So this was not a prerequisite for the old functionality.  With these changes, does running with --new-revision still work even if the python stuff has not been built?  If not, it's going to break people.